commands/rhyme_command.rb

```diff
@@ -0,0 +1,50 @@
+module Commands
+  class RhymeCommand
+    class << self
+      include Discordrb::Webhooks
+
+      def name
+        :rhyme
+      end
+
+      def attributes
+        {
+          min_args: 1,
+          usage: 'rhyme term',
+          description: 'Return a list of rhymes for a term.'
+        }
+      end
+
+      def command(event, *term)
+        return if event.from_bot?
+
+        request = 'https://api.datamuse.com/words?rel_rhy=' << term[0]
+        response = RestClient.get(request)
+        body = JSON.parse response
+
+        groups = body.group_by { |word| word['numSyllables'] }
+
+        if groups.empty?
+          event << 'No synonyms for the word ' << term[0] << ' were found'
+        end
+
+        groups_sorted = groups.sort_by { |num_syllables| num_syllables }
+        embed_fields = groups_sorted.map do |(num_syllables, syllables)|
```

Comment:
Following the previous suggestion, this would be 

```suggestion
        embed_fields = groups.map do |(num_syllables, syllables)|
```